cookiecutter/hooks/post_gen_project.py

```diff
@@ -61,7 +61,7 @@ def main() -> None:
     )
     print()
     print(f"cd {cookiecutter.github_repo_name}")
-    print("python3 -m venv .venv")
+    print("python3.11 -m venv .venv")
```

Comment:
:thinking: 

```
python --version
Python 2.7.18
```

Reply:
Even with python 2 de-installed, a simple `python` binary doesn't exist for me

Reply:
@Marenz I think many use `python-is-python3` package but better to use `python3` in the instructions.

Reply:
> It is a bit crazy that Tumbleweed, which is supposed to have the latest software as soon as they are released, still has python2. :o

Oh it has python3 as well, including 3.11, it just also has python2 ;)

> @Marenz I think many use python-is-python3 package but better to use python3 in the instructions.

That doesn't seem to exist on tumbleweed

Here is a thread discussing exactly this actually: https://www.reddit.com/r/openSUSE/comments/q37bda/comment/hfqnmnz/?utm_source=reddit&utm_medium=web2x&context=3

Reply:
Interesting, sounds like we have to keep it python3 then.

Reply:
So, leaving `python3`, @cwasicki do you plan to update this PR having my suggestion in mind?

Reply:
As I said, I think it is better not to, because this is only a temporary issue because Python 3.11 is not available by default in all distros, but it will be soon enough, and if we hardcore 3.11 here we'll be encouraging users to use an old Python version.

Reply:
Oh, I just meant hard-coding the version in the comment in case this requirement could be relaxed in future. Anyway, your merge answered that already.

Reply:
Currently the pyproject.toml hardcodes 3.11 as min requirement.  We could turn that a template variable filled from a constant.  Or maybe even ask users if they want a newer version.  Then the script can use the constant to print instructions as well.

Does cookiecutter already check the min requirements are met?  Because the pyproject.toml of the top level project also mentions 3.11.  Does that mean we can't run cookiecutter with older versions of python?